src/components/quote.js

```diff
@@ -0,0 +1,64 @@
+import '../App.css';
+import React, { useState } from 'react';
+
+function Quote() {
+  const [quote, setQuote] = useState('');
+  const [author, setAuthor] = useState('');
+  const [loading, setLoading] = useState(false);
+
+  const category = 'success';
+
+  const getQuotes = async () => {
+    const url = `https://api.api-ninjas.com/v1/quotes?category=${category}`;
+    try {
+      setLoading(true);
+      fetch(url, {
+        method: 'GET',
+        withCredentials: true,
+        headers: {
+          'X-Api-Key': '4zb2YJzIg2E3oaKCJJP6xA==LwAtU3SF0SIbvgeq',
+          'Content-Type': 'application/json',
+        },
+      })
+        .then((response) => response.json())
+        .then((data) => {
+          setQuote(data[0].quote);
+          setAuthor(data[0].author);
+        });
+      setLoading(false);
+    } catch (error) {
+      setLoading(false);
+      // print error to console for debugging purposes
```

Comment:
- [OPTIONAL] Kindly make sure to remove the comments in order to keep your codebase clean and professional.